docs/api/RESTAPI.md

```diff
@@ -57,7 +57,7 @@ Submits a workflow to Cromwell. Note that this endpoint can accept an unlimited
 |**FormData**|**workflowInputs_5**  <br>*optional*|A fifth JSON or YAML file containing inputs.|file||
 |**FormData**|**workflowOptions**  <br>*optional*|JSON file containing configuration options for the execution of this workflow.|file||
 |**FormData**|**workflowSource**  <br>*required*|The workflow source file to submit for execution.|file||
-|**FormData**|**workflowType**  <br>*optional*|The workflow language for the file you submitted. Cromwell currently supports WDL.|enum (wdl, cwl)|`"wdl"`|
+|**FormData**|**workflowType**  <br>*optional*|The workflow language for the file you submitted. Cromwell currently supports WDL.|string||
```

Comment:
looks like some stowaways from other people's doc-changing events

Reply:
ToL: I'm fine leaving this non-automated as long as reviewers are checking on each other / it doesn't really matter when we re-run the generator. If we want to make this even more automated in some way, that's always an option too. Alternatively, we could also have folks quickly scan a PR [checklist](https://www.npr.org/2017/10/30/559996276/the-trick-to-surviving-a-high-stakes-high-pressure-job-try-a-checklist).

Reply:
I'd love to see it automated (I was kind of hoping it already was) and personally woudln't mind a checklist, but checklists have been soundly rejected whenever they've come up